packages/react/src/components/ComboBox/ComboBox.tsx

```diff
@@ -211,6 +212,29 @@ interface IComboBoxInternalProps extends Omit<IComboBoxProps, 'ref'> {
   };
 }
 
+/**
+ * Depth-first search to find the first descendant element where the match function returns true.
+ * @param element - element to start searching at
+ * @param match - the function that determines if the element is a match
+ * @returns the matched element or null no match was found
+ */
+function findFirstDescendant(element: HTMLElement, match: (element: HTMLElement) => boolean): HTMLElement | null {
+  const children = getChildren(element);
+
+  // For loop is used because forEach cannot be stopped.
+  for (let index = 0; index < children.length; index++) {
+    const child = children[index];
+    if (match(children[index])) {
```

Comment:
You should be able to use `child` rather than `children[index]` here and on the next line unless I'm missing something.

Reply:
doh!